src/test/java/org/assertj/core/internal/maps/Maps_assertAnySatisfyingConsumer_Test.java

```diff
@@ -0,0 +1,89 @@
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2018 the original author or authors.
+ */
+package org.assertj.core.internal.maps;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.Assertions.assertThatNullPointerException;
+import static org.assertj.core.data.MapEntry.entry;
+import static org.assertj.core.error.ElementsShouldSatisfy.elementsShouldSatisfyAny;
+import static org.assertj.core.test.Maps.mapOf;
+import static org.assertj.core.test.TestData.someInfo;
+import static org.assertj.core.test.TestFailures.failBecauseExpectedAssertionErrorWasNotThrown;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+import static org.mockito.Mockito.verify;
+
+import java.util.Map;
+
+import org.assertj.core.internal.MapsBaseTest;
+import org.assertj.core.test.Player;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class Maps_assertAnySatisfyingConsumer_Test extends MapsBaseTest {
+
+  private Map<String, Player> greatPlayers;
+
+  @Override
+  @BeforeEach
+  public void setUp() {
+    super.setUp();
+    greatPlayers = mapOf(entry("Bulls", jordan), entry("Spurs", duncan), entry("Lakers", magic));
+  }
+
+  @Test
+  public void should_pass_if_one_entry_satisfies_the_given_requirements() {
+    maps.assertAnySatisfy(someInfo(), greatPlayers, (team, player) -> {
+      assertThat(team).isIn("Lakers");
+      assertThat(player.getPointsPerGame()).isGreaterThan(18);
+    });
+  }
+
+  @Test
+  public void should_fail_if_the_map_under_test_is_empty_whatever_the_assertions_requirements_are() {
+    actual.clear();
+    try {
+      maps.assertAnySatisfy(someInfo(), actual, ($1,$2) -> assertThat(true).isTrue());
+    } catch (AssertionError e) {
+      verify(failures).failure(info, elementsShouldSatisfyAny(actual));
+      return;
+    }
+    failBecauseExpectedAssertionErrorWasNotThrown();
+  }
+
+  @Test
+  public void should_fail_if_no_entry_satisfies_the_given_requirements() {
+    try {
+      maps.assertAnySatisfy(someInfo(), actual, ($1, $2) -> {
+        assertThat(true).isFalse();
+      });
+    } catch (AssertionError e) {
+      verify(failures).failure(info, elementsShouldSatisfyAny(actual));
+      return;
+    }
+    failBecauseExpectedAssertionErrorWasNotThrown();
+  }
+
+  @Test
+  public void should_fail_if_actual_is_null() {
+    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> maps.assertAnySatisfy(someInfo(), null, (team, player) -> {
```

Comment:
either use `expectAssertionError` or static import `AssertionsUtil.assertThatAssertionErrorIsThrownBy`which is a shortcut to `assertThatExceptionOfType(AssertionError.class).isThrownBy`